3RD-PARTY-NOTICES.md

```diff
@@ -71,388 +71,374 @@ _2022-01-26 11:04:45 CET_
 > - **Embedded license files**: [jackson-dataformat-yaml-2.11.2.jar/META-INF/LICENSE](jackson-dataformat-yaml-2.11.2.jar/META-INF/LICENSE) 
     - [jackson-dataformat-yaml-2.11.2.jar/META-INF/NOTICE](jackson-dataformat-yaml-2.11.2.jar/META-INF/NOTICE)
 
-**14** **Group:** `com.google.code.findbugs` **Name:** `jsr305` **Version:** `3.0.1` 
+**14** **Group:** `com.github.ben-manes.caffeine` **Name:** `caffeine` **Version:** `3.0.5` 
 > - **Manifest License**: Apache License, Version 2.0 (Not Packaged)
-> - **POM Project URL**: [http://findbugs.sourceforge.net/](http://findbugs.sourceforge.net/)
+> - **POM Project URL**: [https://github.com/ben-manes/caffeine](https://github.com/ben-manes/caffeine)
 > - **POM License**: Apache License, Version 2.0 - [https://www.apache.org/licenses/LICENSE-2.0](https://www.apache.org/licenses/LICENSE-2.0)
+> - **Embedded license files**: [caffeine-3.0.5.jar/META-INF/LICENSE](caffeine-3.0.5.jar/META-INF/LICENSE)
 
-**15** **Group:** `com.google.errorprone` **Name:** `error_prone_annotations` **Version:** `2.3.4` 
+**15** **Group:** `com.google.code.findbugs` **Name:** `jsr305` **Version:** `3.0.1` 
+> - **Manifest License**: Apache License, Version 2.0 (Not Packaged)
+> - **POM Project URL**: [http://findbugs.sourceforge.net/](http://findbugs.sourceforge.net/)
 > - **POM License**: Apache License, Version 2.0 - [https://www.apache.org/licenses/LICENSE-2.0](https://www.apache.org/licenses/LICENSE-2.0)
 
-**16** **Group:** `com.google.guava` **Name:** `failureaccess` **Version:** `1.0.1` 
-> - **Manifest Project URL**: [https://github.com/google/guava/](https://github.com/google/guava/)
-> - **Manifest License**: Apache License, Version 2.0 (Not Packaged)
+**16** **Group:** `com.google.errorprone` **Name:** `error_prone_annotations` **Version:** `2.1.3` 
 > - **POM License**: Apache License, Version 2.0 - [https://www.apache.org/licenses/LICENSE-2.0](https://www.apache.org/licenses/LICENSE-2.0)
 
-**17** **Group:** `com.google.guava` **Name:** `guava` **Version:** `30.0-jre` 
+**17** **Group:** `com.google.guava` **Name:** `guava` **Version:** `24.1.1-jre` 
```

Comment:
@mkleen: You recently worked on the Guava topic, right? Do you know if the downgrade outlined here was intended?

Reply:
We recently removed guava as dependency from CrateDB. https://github.com/crate/crate/pull/12057
This is transitive dependency introduced by the azure discovery plugin, therefore the downgrade makes sense.
```
+--- project :plugins:es-repository-azure
|    +--- project :server (*)
|    +--- com.microsoft.azure:azure-storage:8.6.6
|    |    +--- com.fasterxml.jackson.core:jackson-core:2.9.4 -> 2.11.2
|    |    +--- org.slf4j:slf4j-api:1.7.12 -> 1.6.2
|    |    +--- org.apache.commons:commons-lang3:3.4 -> 3.5
|    |    \--- com.microsoft.azure:azure-keyvault-core:1.2.4
|    |         +--- org.apache.commons:commons-lang3:3.8.1 -> 3.5
|    |         \--- com.google.guava:guava:24.1.1-jre
|    |              +--- com.google.code.findbugs:jsr305:1.3.9 -> 3.0.1
|    |              +--- com.google.errorprone:error_prone_annotations:2.1.3
|    |              +--- com.google.j2objc:j2objc-annotations:1.1
|    |              \--- org.codehaus.mojo:animal-sniffer-annotations:1.14
|    \--- com.microsoft.azure:azure-keyvault-core:1.2.4 (*)
```